test/CMakeLists.txt

```diff
@@ -1,44 +1,45 @@
 set(SRCROOT "${PROJECT_SOURCE_DIR}/test/src")
 
-include_directories("${PROJECT_SOURCE_DIR}/include")
-include_directories("${PROJECT_SOURCE_DIR}/extlibs/headers")
-include_directories("${SRCROOT}/TestUtilities")
+add_library(sfml-test-main "${SRCROOT}/CatchMain.cpp")
```

Comment:
You may want to make this

`add_library(sfml-test-main STATIC "${SRCROOT}/CatchMain.cpp")`

Reply:
That makes sense. No need to build a DLL out of this.

Reply:
Fixed.